packages/integrations/markdoc/src/index.ts

```diff
@@ -233,12 +233,11 @@ export const Content = createComponent({
 			},
 			'astro:server:setup': async ({ server }) => {
 				server.watcher.on('all', (event, entry) => {
-					if (prependForwardSlash(pathToFileURL(entry).pathname) === markdocConfigResultId) {
-						console.log(
-							yellow(
-								`${bold('[Markdoc]')} Restart the dev server for config changes to take effect.`
-							)
-						);
+					for (const markdocConfigBase of SUPPORTED_MARKDOC_CONFIG_FILES) {
+						const absolutePath = new URL(markdocConfigBase, astroConfig.root);
+						if (entry === fileURLToPath(absolutePath)) {
```

Comment:
Not blocking the PR, but perhaps this can be optimized as 

```js
if (SUPPORTED_MARKDOC_CONFIG_FILES.some(f => entry.endsWith(s))) {
  server.restart()
}
```

(probably good enough) as this function would be triggered repeatedly for any file changes.

Reply:
Ah, was needlessly worried about a markdoc config located in the wrong place. This is probably fine!

Reply:
Nice!